core/src/avm2/globals/Date.as

```diff
@@ -1,5 +1,461 @@
+        }
+        public function set hours(value:Number):void {
+            this.AS3::setHours(value);
+        }
+
+        public function get minutes():Number {
+            return this.AS3::getMinutes();
+        }
+        public function set minutes(value:Number):void {
+            this.AS3::setMinutes(value);
+        }
+
+        public function get seconds():Number {
+            return this.AS3::getSeconds();
+        }
+        public function set seconds(value:Number):void {
+            this.AS3::setSeconds(value);
+        }
+
+        public function get milliseconds():Number {
+            return this.AS3::getMilliseconds();
+        }
+        public function set milliseconds(value:Number):void {
+            this.AS3::setMilliseconds(value);
+        }
+
+        public function get fullYearUTC():Number {
+            return this.AS3::getUTCFullYear();
+        }
+        public function set fullYearUTC(value:Number):void {
+            this.AS3::setUTCFullYear(value);
+        }
+
+        public function get monthUTC():Number {
+            return this.AS3::getUTCMonth();
+        }
+        public function set monthUTC(value:Number):void {
+            this.AS3::setUTCMonth(value);
+        }
+
+        public function get dateUTC():Number {
+            return this.AS3::getUTCDate();
+        }
+        public function set dateUTC(value:Number):void {
+            this.AS3::setUTCDate(value);
+        }
+
+        public function get hoursUTC():Number {
+            return this.AS3::getUTCHours();
+        }
+        public function set hoursUTC(value:Number):void {
+            this.AS3::setUTCHours(value);
+        }
+
+        public function get minutesUTC():Number {
+            return this.AS3::getUTCMinutes();
+        }
+        public function set minutesUTC(value:Number):void {
+            this.AS3::setUTCMinutes(value);
+        }
+
+        public function get secondsUTC():Number {
+            return this.AS3::getUTCSeconds();
+        }
+        public function set secondsUTC(value:Number):void {
+            this.AS3::setUTCSeconds(value);
+        }
+
+        public function get millisecondsUTC():Number {
+            return this.AS3::getUTCMilliseconds();
+        }
+        public function set millisecondsUTC(value:Number):void {
+            this.AS3::setUTCMilliseconds(value);
+        }
+
+        public function get time():Number {
+            return this.AS3::getTime();
+        }
+        public function set time(value:Number):void {
+            this.AS3::setTime(value);
+        }
+
+        public function get timezoneOffset():Number {
+            return this.AS3::getTimezoneOffset();
+        }
+
+        public function get day():Number {
+            return this.AS3::getDay();
+        }
+
+        public function get dayUTC():Number {
+            return this.AS3::getUTCDay();
+        }
+
+
+        prototype.setTime = function(t:* = undefined):Number {
```

Comment:
I think we'd prefer to have prototype methods defined in the class initializer at the top of the class to match FFDEC and the rest of the globals